src/components/JobTable/NextRun.test.js

```diff
@@ -1,24 +1,50 @@
 import React from 'react'
-import { shallow } from 'enzyme'
+import { mount, shallow } from 'enzyme'
+import { Tooltip } from '@dhis2/ui'
 import NextRun from './NextRun'
 
 // Z is the zone designator for the zero UTC offset
 const now = new Date('2010-10-10T10:10:10.000Z').valueOf()
 
-// The server does not specify a timezone, but is assumed to provide UTC
 const past = '2009-10-10T10:10:10.000'
 const future = '2011-10-10T10:10:10.000'
+const timeZone = Intl.DateTimeFormat().resolvedOptions().timeZone
+
+// in this test, we are only passing timestamps to the hook (and there is no server/client time diff)
+// so mock getServerZonedISOString by returning the original timestamp
```

Comment:
Have we adequately covered all expected scenarios in this test? Should we not also test for the expected behaviour in case there is a difference between the client and server TZ?

Reply:
I thought about that originally but then decided against adding tests for different time zones. I guess I thought it would be a bit counter to the goals of the hook, i.e. we want to provide a hook that abstracts away the logic (and responsibility) for adjusting for client/server time zones. 

Hence, I figured that the tests in app-runtime should be sufficient for testing those differences and then a consuming app should only be responsible for testing for a given time zone / testing the "base" functionality without worrying about the client/server time zone difference?

Reply:
Yeah I agree, that it's not really necessary to test external library functionality. From the test coverage report it also seems like we've covered everything in NextRun so we should be good 👍